ckan/setup/server_start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+if test -f "$DIR/.env"; then
+    set -o allexport; source $DIR/.env; set +o allexport
+fi
+
+# Run web application
+$CKAN_PY_ENV/newrelic-admin run-program $CKAN_PY_ENV/gunicorn --worker-class gevent --paste $CKAN_INI "$@"
```

Comment:
Instead of CKAN_PY_ENV, how about we just assume the caller has already setup the path correctly and/or activated the virtualenv?

Reply:
I hit a blocker due to [not being able to setup the virtual environment via Ansible](https://stackoverflow.com/questions/20040141/ansible-command-from-inside-virtualenv). I could [setup the path variable in supervisor](https://stackoverflow.com/questions/15202760/how-to-use-virtualenvwrapper-in-supervisor) to contain the virtualenv, is that a reasonable solution?

Reply:
I went ahead and removed that variable/path and we can discuss changes/details in the [datagov-deploy PR](https://github.com/GSA/datagov-deploy/pull/2317)